src/borg/item.pyx

```diff
@@ -652,9 +653,11 @@ class ItemDiff:
         if self._item1.is_fifo() or self._item2.is_fifo():
             changes.append(self._presence_diff('fifo'))
 
-        if not (self._item1.get('deleted') or self._item2.get('deleted')):
-            changes.append(self._owner_diff())
-            changes.append(self._mode_diff())
+        if not self._content_only:
+            if not (self._item1.get('deleted') or self._item2.get('deleted')):
+                changes.append(self._owner_diff())
+                changes.append(self._mode_diff())
+            changes.extend(self._time_diffs())
```

Comment:
Yeah, pondered on that for a while and decided to leave that in. It makes it so that we can get the time of deletion/recreation. Do you think we shouldn't do it if either one is deleted?

Reply:
well, the deleted item has nothing inside because there was nothing in the filesystem.
so we can't see the time of deletion, only the time of creation IF one compares the right archives or the fs supports creation time.

but one could see the timestamps of the other item.
as long as this is displayed in a clear way we could keep it, but it is somehow similar to owner and mode besides the aspect of timing.

Reply:
isn't 751 False and the function call not generating any output if one of the items is deleted?

if that is the case, 661 could be also indented to not get executed if one item  is deleted.

Reply:
Right, in that case it would be better to only call it inside the if clause.